converters/toZigbee.js

```diff
@@ -2130,7 +2139,7 @@ const converters = {
             if (key === 'state') {
                 await entity.read('genOnOff', ['onOff']);
             } else if (key === 'color') {
-                await entity.read('lightingColorCtrl', ['currentHue', 'currentSaturation']);
+                await entity.read('lightingColorCtrl', ['currenthHue', 'currentSaturation']);
```

Comment:
Are you sure this change is correct?

Reply:
yes, it had been merged some hours ago already. https://github.com/Koenkk/zigbee-herdsman-converters/pull/1450

Reply:
Sorry, reintroduced that although I PRed this some hours ago... https://github.com/Koenkk/zigbee-herdsman-converters/pull/1450